docs/docson/build-schema.json

```diff
@@ -292,6 +319,10 @@
             "type" : "boolean",
             "description": "(experimental currently)default true(Set use OCaml stdlib or not)"
         },
+        "buildable-targets": {
+            "$ref": "#/definitions/buildable-targets",
```

Comment:
Oh I misunderstood  and thought you wanted `type buildableTargets = Js | targets list` with `type targets = Native of string | Bytecode of string`.
I can update this to be `type buildableTargets = targets list` with `type targets = Js | Native of string | Bytecode of string` if that's what you want. Btw it defaults to Js, so none of the bsconfig.json need to be updated.